gradle.properties

```diff
@@ -1,3 +0,0 @@
-org.gradle.daemon=true
```

Comment:
The gradle daemon caches oft repeated build instructions. Great for speedups. Android studio already has a daemon by default. Command line doesn't and needs this file to tell it to use a gradle daemon.

* http://jimulabs.com/2014/10/speeding-gradle-builds/